lib/hawkular/metrics/metric_api.rb

```diff
@@ -152,8 +171,14 @@ def encode_params(params)
       private
 
       def get_data_helper(id, params)
-        resp = @client.http_get("/#{@resource}/#{ERB::Util.url_encode(id)}/data/?" +
-          encode_params(params))
+        path = "/#{@resource}/#{ERB::Util.url_encode(id)}/"
+        if @new_api_supported
```

Comment:
I'm not sure if is appropriate to use elvis operator here because inside I have an elvis operator too if I use an elvis operator in this part, I'll have nested elvis operators. I don't think it will make the code readable.